telemetry/v1.proto

```diff
@@ -0,0 +1,809 @@
+// Important: This schema is currently in BETA
+
+syntax = "proto3";
+
+import "google/protobuf/any.proto";
+import "google/protobuf/timestamp.proto";
+
+option objc_class_prefix = "SNTPB";
+
+package santa.telemetry.v1;
```

Comment:
I don't think this is a safe change, we should chat.

Reply:
As discussed - this **is** a breaking change but will not be included until the 2024.11 release at the earliest.